README.md

```diff
@@ -19,6 +19,7 @@ Also check [Before you install](https://docs.bigbluebutton.org/2.2/install.html#
 | `bbb_apt_mirror` | apt repo server for BigBlueButton packages | `https://ubuntu.bigbluebutton.org` | other value would be e.g. `https://packages-eu.bigbluebutton.org` |
 | `bbb_letsencrypt_enable` | Enable letsencrypt/HTTPS | `yes` |
 | `bbb_letsencrypt_email` | E-mail for use with letsencrypt | | _(required when letsencrypt is enabled)_ |
+| `bbb_letsencrypt_staging` | Genereate test (staging) letsencrypt certificates | `false` |
```

Comment:
Minor README Issue missing a Comment `| My Comment |`

Reply:
Thank you for reviewing the PR.
Resolved in: https://github.com/ebbba-org/ansible-role-bigbluebutton/pull/106/commits/42cdd9a7be491680bcc31df9fbf8ae296fd9f335